srcpkgs/cquery/template

```diff
@@ -0,0 +1,29 @@
+pkgname="cquery"
+version="v20180302"
+revision=1
+wrksrc="${pkgname}"
+build_style=waf3
+configure_args="--llvm-config=llvm-config"
+hostmakedepends="git"
+makedepends="clang llvm"
+depends="clang"
+short_desc="Low-latency language server for C, C++, and Objective-C"
+maintainer="jtalowell <jtalowell@protonmail.com>"
+license="MIT"
+homepage="https://github.com/cquery-project/cquery/"
+
+do_fetch() {
```

Comment:
we don't accept that. use a tagged release

Reply:
I'm using a tagged release. I checkout the tagged release in `do_fetch`: `git checkout ${version}`.

The current release won't build without error due to usage of the deprecated `-latomic` flag for `ld` so I've checked out the release using git but build using the updated build script. This won't be an issue with the next release.

Is there alternative way you recommend handling this?

Reply:
the way that is present on most templates

distfiles="https://github.com/cquery-project/cquery/archive/v${version}.tar.gz"
checksum=

Reply:
> The current release won't build without error due to usage of the deprecated -latomic flag for ld so I've checked out the release using git but build using the updated build script. This won't be an issue with the next release.

add a patch to srcpkgs/cquery/patches/\<name>.patch that fixes it.